app/controllers/admins/public_servants_controller.rb

```diff
@@ -75,10 +87,16 @@ def public_servant_params
 
     params
       .require(:admin)
-      .permit(:name, :email, :record_number, :dependency, :administrative_unit, :charge, :surname, :second_surname, :is_observer)
+      .permit(:name, :email, :record_number, :dependency, :administrative_unit, :charge, :surname, :second_surname, :is_observer,:is_comptroller)
       .merge(services: services, is_public_servant: true)
   end
 
+  def comptroller_params
+    params
+      .require(:admin)
+      .permit(:name, :email, :record_number, :dependency, :administrative_unit, :charge, :surname, :second_surname,:is_comptroller)
```

Comment:
Line is too long. [131/80]<br>Place the . on the previous line, together with the method call receiver.<br>Space missing after comma.